### ISSUE TITLE
Inventory Download

### ISSUE THREAD

**Issue by markfaine** [ansible-semaphore/semaphore#590]
We keep our inventory in Gitlab so I was having trouble figuring out how to reference it in semaphore.  Do you think it would work, to create a playbook  for the purpose of downloading our inventory, and then configure an inventory file that points to that inventory?

**Comment by strangeman** [comment]
Sure, why not. You can download an inventory to the server and then just point to in with `-i` flag of `ansible-playbook`.

**Comment by fiftin** [comment]
@markfaine Are you keep your inventory separate from playbooks?

**Comment by markfaine** [comment]
Yes, it's in it's own repo for now though are looking for ways to create a dynamic inventory.
